frontend/packages/console-demo-plugin/package.json

```diff
@@ -8,6 +8,12 @@
     "@console/shared": "0.0.0-fixed"
   },
   "consolePlugin": {
-    "entry": "src/plugin.tsx"
+    "entry": "src/plugin.tsx",
+    "integrationTestSuites": {
```

Comment:
@christianvogt @spadgett Would you prefer this to be shortened to `integrationTests` ?

This is the plugin metadata from which Protractor `suites` object `{[suiteName: string]: string[]}` is created and applied to `config.suites` in `integration-tests/protractor.conf.ts`.

Reply:
This is fine with me.